.github/pull-request-template.md

```diff
@@ -1,9 +1,8 @@
 **Acceptance criteria:**  
-- [ ] PR title includes jira/trello ticket reference
-- [ ] Updated Postman/Swagger
+- [ ] PR title includes ticket reference
+- [ ] Updated API spec
 - [ ] Updated additional documentation (please add a link if other than README 😊)
 - [ ] Includes tests
```

Comment:
Not sure about the exact wording, but I think we should add a checklist for adding env vars to the server and test env file, as a lot tends to forget it 😅 
```suggestion
- [ ] Includes tests
- [ ] New environment variables are added to the server and other places where applicable
```